xbmc/FileItem.cpp

```diff
@@ -3578,11 +3578,44 @@ std::string CFileItem::GetBaseMoviePath(bool bUseFolderNames) const
 
       URIUtils::GetParentPath(name2, strMovieName);
     }
+
+    // Remove 'Disc' subdirectory so actual title passed to scraper
+    if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+            CSettings::SETTING_MYVIDEOS_HANDLEDISCN))
+    {
+      size_t iDisc = GetDiscPath(strMovieName);
```

Comment:
`const size_t ...`

Please remove the `i`prefix from the variable's name (coding guideline).